toolkit/tools/internal/file/file.go

```diff
@@ -320,7 +320,7 @@ func CopyResourceFile(srcFS fs.FS, srcFile, dst string, dirmode os.FileMode, fil
 	}
 	defer source.Close()
 
-	destination, err := os.OpenFile(dst, os.O_WRONLY|os.O_CREATE, filemode)
+	destination, err := os.OpenFile(dst, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, filemode)
```

Comment:
Could you also explicitly mention that we'll overwrite existing files in the comment for this function?

Reply:
Done.